src/db.js

```
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '12ND1205',
```

Avoid hardcoding a default database password in source. Instead rely on the environment variable (or fallback to an empty value) and document required credentials; in production you may want to throw an error if sensitive vars are missing. Hardcoded secrets risk accidental leakage and make environments inconsistent.